frontend/package.json

```diff
@@ -94,7 +94,8 @@
     "clean:emotion-types": "rimraf node_modules/@emotion/core/types",
     "postinstall": "patch-package && npm run gen:theme-typings && npm run clean:emotion-types && npm --prefix ../shared install",
     "start": "env-cmd -f .buildtime-env craco --openssl-legacy-provider start",
-    "build": "cross-env CI=false BUILD_PATH='../dist/frontend' INLINE_RUNTIME_CHUNK=false SKIP_PREFLIGHT_CHECK=true craco build",
+    "build": "npm run build:dd-chunk && cross-env CI=false BUILD_PATH='../dist/frontend' INLINE_RUNTIME_CHUNK=false SKIP_PREFLIGHT_CHECK=true craco build && cd ../ && cp -rf frontend/static/js dist/frontend/static",
+    "build:dd-chunk": "webpack --config webpack.config.js",
```

Comment:
/nit: since webpack is dd-specific, let's have `dd` in the config file as well (just like for the tsconfig file), maybe like `webpack.dd.config.js`

Reply:
[51264f3](https://github.com/opengovsg/FormSG/pull/5571/commits/51264f33c217b8fc73fb25254e5bf39cfef90ed2)